content/blog/2020/09/17/Equality-of-Being-Talk-Untitled-Festival.md

```diff
@@ -0,0 +1,20 @@
+---
+Title: Liam Kavanagh Equality of Being Talk at Untitled Festival
+Created: 2020-09-17
+Authors: [rufuspollock]
+---
+
+As part of day 1 of the Untitled Festival, held on 17th & 18th September 2020 online, Liam Kavanagh presented for open discussion the topic of Equality of Being.
+
+This talk explored the implications of understanding equality intellectually, and introduced the wider topic of the Equality Complex.
+
+Below you can watch the Equality of Being talk from day 1 of the festival.
+
+For more information on the Untitled Festival and the wider Politics of Being discussions please visit [here](https://lifeitself.org/blog/2020/09/17/politics-of-being-talk-untitled-festival).
+
+You can learn more about the Equality Complex [here](https://lifeitself.org/blog/2019/12/01/blind-spot-3-the-equality-complex).
```

Comment:
remember we don't link "here" because SEO is better with a full phrase. So have:

```
You can [learn more about the Equality Complex here](https://lifeitself.org/blog/2019/12/01/blind-spot-3-the-equality-complex).
```

or

```
You can learn more about the [Equality Complex here](https://lifeitself.org/blog/2019/12/01/blind-spot-3-the-equality-complex).
```